.github/workflows/reusable-auto-update-kubebuilder.yml

```
@@ -0,0 +1,94 @@
+name: Auto Update Kubebuilder Scaffolding
+
+# The 'kubebuilder alpha update 'command requires write access to the repository to create a branch
+# with the update files and allow you to open a pull request using the link provided in the issue.
+# The branch created will be named in the format kubebuilder-update-from-<from-version>-to-<to-version> by default.
+# To protect your codebase, please ensure that you have branch protection rules configured for your 
+# main branches. This will guarantee that no one can bypass a review and push directly to a branch like 'main'.
+permissions:
+  contents: write
+  issues: write
+  models: read
```

actionlint (1.7.7)

unknown permission scope "models". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)